### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by koyakei**

### CODE REVIEW

File: ja/orm/entities.rst [hunk 1]
```diff
@@ -235,18 +235,18 @@ CakeのORMを使うためにエンティティークラスを生成する必要
 
 .. _entities-mass-assignment:
 
-Mass Assignment（一気に代入）
+マスアサインメント( Mass Assignment )
 ===============
```
Comment (markstory) [inline ja/orm/entities.rst]: Title underline needs to be extended.

File: ja/orm/entities.rst [hunk 2]
```diff
@@ -318,14 +318,14 @@ mass-assignment が保護されたフィールドへのアクセスすること
 -----------------------------------
 
 データベスに既に存在する行をエンティティーが表しているのか知る必要がある時があります。
-``isNew()`` でそれが分かります。::
+``isNew()`` でそれが分かります。:
```
Comment (markstory) [inline ja/orm/entities.rst]: The `::` enable code highlighting and should probably stay.

File: ja/orm/entities.rst [hunk 3]
```diff
@@ -333,12 +333,12 @@ mass-assignment が保護されたフィールドへのアクセスすること
 
 .. _lazy-load-associations:
 
-Lazy Loading Associations
+アソシエーションのLazy ローディング
 =========================
```
Comment (markstory) [inline ja/orm/entities.rst]: Underline length.

File: ja/orm/entities.rst [hunk 4]
```diff
@@ -471,7 +471,7 @@ JSON/配列フォーマットで出力したくないフィールドがある場
 
     }
 
-このリストは、 ``hiddenProperties`` で編集できます。::
+このリストは、 ``hiddenProperties`` で編集できます。:
```
Comment (markstory) [inline ja/orm/entities.rst]: Those `::` need to stay.

File: ja/orm/entities.rst (hunk 2 above)
Comment (koyakei) [inline ja/orm/entities.rst]: https://www.facebook.com/wataru.terada
said in japanese do not need it.

Comment (markstory) [inline ja/orm/entities.rst]: Its only not necessary if there is a `::` on a line below. For example,

```rst
Some words::

    $this->is->code();

More words

and 

```rst
Some words

::

    $this->is->code();

More words
```

Will both correctly generate a formatted code block. But what you have right now will not. If you want to test the build output locally. The README has instructions on how to install the dependencies and run sphinx.

**Comment by antograssiot** [comment]
@koyakei it seems that this PR can't be merged anymore. Should we close it in favor of #3024 or will you update it ?

**Comment by koyakei** [comment]
please close it  thanks.
In JPN translation group made good solution of it.


2015-08-04 1:03 GMT+09:00, Anto <notifications@github.com>:
> @koyakei it seems that this PR can't be merged anymore. Should we close it
> in favor of #3024 or will you update it ?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cakephp/docs/pull/3017#issuecomment-127288324


-- 
小柳圭輔 Koyanagi Keisuke <http://srns.strikingly.com/>
Skype twitter <https://twitter.com/koyakei> facebook
<https://www.facebook.com/koyakei>  = koyakei

**Pull request closed without merging**